json-csharp/GetValueByKeyFromJObject/Tests/JObjectManipulationUnitTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using GetValueByKeyFromJObject;
+
+namespace Tests
+{
+    public class JObjectManipulationUnitTest
+    {
+        private readonly JObjectManipulation _jObjectManipulation;
+        public JObjectManipulationUnitTest()
```

Comment:
Separate private and public members with a line.

Reply:
Same as this